src/integration-tests/tcp/freedom-module.ts

```diff
@@ -46,6 +46,113 @@ parentModule.on('listen', () => {
   });
 });
 
+parentModule.on('floodtwoclient', (data) => {
```

Comment:
Add a comment with the command line arguments you might want to use to run this; e.g. using your server, nc or such.